.github/workflows/verify-markdown-formatting.yml

```diff
@@ -0,0 +1,18 @@
+name: "Verify Markdown formatting"
+
+on:
+  push:
+    paths:
+      - "**.md"
+  pull_request:
+    paths:
+      - "**.md"
+
+jobs:
+  stylecheck:
+    steps:
+      - name: "Checkout code"
+        uses: actions/checkout@v2
+
+      - name: "Verify Markdown formatting"
+        run: npx prettier@2.0.4 --check "**/*.md"
```

Comment:
@SaschaMann Would you be okay with me adding the "only check PR files" functionality in a different PR to get this merged?

Reply:
Yea, sure